packages/graph/hash_graph/README.md

```diff
@@ -1,5 +1,33 @@
 # The HASH Graph API
 
+## Run the Graph
+
+The easiest way to run the Graph API is to use docker. Either invoke docker directly:
+
+```shell
+docker build --tag graph --file ../deployment/graph/Dockerfile .
+docker run --rm --init --network host --name graph --env RUST_LOG=trace --detach graph
+```
+
+or by using `cargo make`:
+
+```shell
+cargo make build-docker --profile production
+cargo make docker-up
+```
+
+The container can be stopped by calling
+
+```shell
+docker stop graph
```

Comment:
While this does stop the container, the container will still be around. Do we want an instruction to stop and remove the container also? that would be something like `docker rm -f graph` I believe.

Reply:
Good point, added that to the readme.